src/toPropertyPath.ts

```diff
@@ -5,5 +5,6 @@ export function toPropertyPath(path: string) {
     .replace(/^(\/|#\/)/, '')
     .split('/')
     .map(decodePointerFragment)
+    .map(fragment => (fragment.indexOf('.') > -1 ? `["${fragment.replace(/"/g, '\\"')}"]` : fragment))
```

Comment:
```suggestion
    .map(fragment => (fragment.includes('.') ? `["${fragment.replace(/"/g, '\\"')}"]` : fragment))
```
Could be a separate function BTW?

Reply:
Done!